src/ui/components/Rating/index.js

```diff
@@ -50,16 +55,19 @@ export class RatingBase extends React.Component {
       const props = {
         className: classNames('Rating-choice', {
           'Rating-selected-star': rating && thisRating <= rating,
+          // Half stars are the result of averages rounded to the nearest
+          // 0.5 place; only show them when readOnly is true.
+          'Rating-half-star': (readOnly && rating && thisRating > rating &&
```

Comment:
I didn't realize it was a float but I think it would still work in non-read only mode. Anyway, yeah, it's not important since I doubt we'll ever let people submit half-star ratings.